examples/README.md

```diff
@@ -5,11 +5,11 @@
 Install and try Manticore in a few shell commands:
 
 ```bash
-# Install system dependencies
-sudo apt-get update && sudo apt-get install python3 python3-dev python3-pip -y
+# (Recommended) Create a virtual environment for Manticore
+mkvirtualenv -p `which python3` mcenv
```

Comment:
`mkvirtualenv` requires `virtualenvwrapper` 3rd party dependency. We should tell people to install that first if we're going to stick with `mkvirtualenv` here

Reply:
Ahhh, good catch. Will fix.